locale/ca/foreman_azure_rm.po

```diff
@@ -0,0 +1,174 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the foreman_azure_rm package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+# Translators:
+# Robert Antoni Buj Gelonch <rbuj@fedoraproject.org>, 2020
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: foreman_azure_rm 2.0.8\n"
```

Comment:
Once I update my gem to a new version other than `2.0.8`, what happens to this file?